superchain/Dockerfile

```diff
@@ -149,11 +149,16 @@ RUN set -eo pipefail
   && chmod -R a+rw ${CARGO_HOME}
 ENV PATH=$PATH:${CARGO_HOME}/bin
 
-# Install Python 3
+# Install Python 3. Use 'pipx' for CLI scripts, so each gets its own venv
+ENV PATH=$PATH:/root/.local/bin
 RUN apt-get update                                                                                                      \
   && apt-get -y install python3 python3-dev python3-pip python3-venv                                                    \
-  && python3 -m pip install --no-input --upgrade pip                                                                    \
-  && python3 -m pip install --no-input --upgrade awscli black setuptools twine wheel                                    \
+  && python3 -m pip install --no-input --upgrade pip setuptools                                                         \
+  && python3 -m pip install pipx                                                                                        \
+  && python3 -m pipx install --system-site-packages awscli                                                              \
+  && python3 -m pipx install --system-site-packages black                                                               \
+  && python3 -m pipx install --system-site-packages twine                                                               \
+  && python3 -m pipx inject --system-site-packages twine wheel                                                          \
```

Comment:
Put these two in the same venv, don't know if this is necessary but *feels* like they need to work together and *maybe* they need to agree on the versions of certain dependencies when they do.

Might be superstition...